brownie/project/main.py

```diff
@@ -781,18 +781,23 @@ def _install_from_ethpm(uri: str) -> str:
 
 def _install_from_github(package_id: str) -> str:
     try:
-        path, version = package_id.split("@")
+        split = package_id.split("@")
+        path = split[0]
+        version = "@".join(split[1:])
         org, repo = path.split("/")
     except ValueError:
         raise ValueError(
             "Invalid package ID. Must be given as [ORG]/[REPO]@[VERSION]"
-            "\ne.g. 'OpenZeppelin/openzeppelin-contracts@v2.5.0'"
+            "\ne.g. 'OpenZeppelin/openzeppelin-contracts@v3.1.0'"
         ) from None
 
     base_install_path = _get_data_folder().joinpath("packages")
     install_path = base_install_path.joinpath(f"{org}")
     install_path.mkdir(exist_ok=True)
-    install_path = install_path.joinpath(f"{repo}@{version}")
+
+    # Sanitize version removing paths For example UMA's ""@uma/core@1.2.0"
+    version_sanitized = version.replace(os.path.sep, '')
```

Comment:
happy to find a better solution to this. Removing the `/` from here, means remapping would not match dependencies.

Reply:
Use of `os.path.sep` here will create inconsistent behaviour between windows and unix based systems.  Brownie converts everything to posix paths, so I think better to just target `/` directly.  Instead of removing, what about replacing with `-`?

Reply:
It's also pretty ugly to have all those extra `@` symbols.  I think it'd be good to also `lstrip("@")`